tests/test_vcompressor.py

```diff
@@ -31,6 +31,62 @@ def test_basic(self):
                 expected_output_file=dup_json_path
             )
 
+    def test_compress_invalid(self):
+        with tempfile.TemporaryDirectory() as tmpdir:
+            cvf_path = os.path.join(tmpdir, "result.cvf")
+            not_exist_path = os.path.join(tmpdir, "do_not_exist.json")
+            result = self.template(
+                ["viztracer", "-o", cvf_path, "--compress", not_exist_path],
+                expected_output_file=None, success=False
+            )
+            self.assertIn("Unable to find file", result.stdout.decode("utf8"))
+
+            result = self.template(
+                ["viztracer", "-o", cvf_path, "--compress", get_json_file_path("fib")],
```

Comment:
You can use a `.py` file in `data` folder I think.

Reply:
Yes, I know what you mean. That's what I wanted to write at first.
But that makes `get_json_file_path("fib.py")` look a little weird.
Maybe I could rename it to `get_tests_data_file_path`, which would be more appropriate?